CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Swift rules cleanup and updates (removed duplicated rules, added [EC602])
+- [#290](https://github.com/green-code-initiative/ecoCode/issues/290) [EC512] Swift port
```

Comment:
why did you add this line in 1.5.1 section and not in unreleased section ?

Reply:
A mistake probably, this PR is 1 month old, I keep rebasing it :)

Reply:
I moved it